transport/transport.go

```diff
@@ -55,6 +62,13 @@ func (r *ResErrTuple) DiscardBody() {
 // returned value's response and error will be passed to client
 type MultipleResponsesHandler func(in <-chan ResErrTuple) ResErrTuple
 
+// Container mapping initialized Transports with http.RoundTripper by transport name
+type Container struct {
```

Comment:
1) Add RoundTrip method with body from backend.RoundTrip as I suggested in storages package
2) Rename it Container is too generic name. Suggestions: Selector, TransportMatcher. Feel free to make your own name which suggests purpose of this struct